sdk-docs/how-to/flask.md

```diff
@@ -0,0 +1,106 @@
+# Add an environment variable
+
+A charm configuration can be added if you require environment variables, for
+example, to pass a token for a service. For example, add the following
+configuration in `charmcraft.yaml`:
+
+```yaml
+config:
+  options:
+    TOKEN:
```

Comment:
Do we recommend upper case config option names? I think PaaS App Charmer will convert from lower case to upper case, but the recommendation is to use lower case in the charm config - see https://juju.is/docs/sdk/styleguide#heading--naming "When naming configuration items or actions, prefer lowercase alphanumeric names, separated with dashes if required".

Reply:
yeh I think this should be lower case. One interesting consequence is the point about dashes since underscore is not the upper case of a dash so if people follow the recommendation around using dashes then their environment variables will have dashes in them when underscores are more common. Any thoughts @gregory-schiano?

The options are:
1. Keep the current behaviour around dashes
1. Plus make available another copy of the environment variables where `-` is replaced with `_`

Reply:
`-` are already replaced by `_`
https://github.com/canonical/paas-app-charmer/blob/main/paas_app_charmer/_gunicorn/charm_state.py#L128